flipperdevices/flipperzero-firmware
# [FL-1619] NFC long APDU emulation

# What's new

- Pull down nfc irqn pin
- Add debug scene for 3 APDU transactions emulation

# Verification 

- Run nfc-list on Pn532 reader
- Send 3 random APDU commands and verify answer reception

# Checklist (do not modify)
